PyRDF/backend/Utils.py

```diff
@@ -1,10 +1,41 @@
-import ROOT
-import os
 import logging
+import multiprocessing
+import os
+import threading
+
+try:
+    import queue
+except ImportError:
+    import Queue as queue
+
+import ROOT
 
 logger = logging.getLogger(__name__)
 
 
+def run_graphs(proxies, numthreads=multiprocessing.cpu_count()):
```

Comment:
Should we keep the naming convention of the original `RunGraphs` ?

Reply:
Not really, the name can be changed. Generally speaking, if this function will be backend-dependent (as you also suggest in the next comment), I think it should still have the same name for all backends. That said, I can think of other options for the name:
* trigger_actions
* trigger_eventloops
* submit_njobs
* submit_ngraphs

There might be even more

Reply:
I would be in favour of keeping the same name, I don't see any reason why we should change it if the functionality is the same (we are already changing where it is accessed). Ideally, in the future, a person using RDF from Python should still use `ROOT.RDF.RunGraphs` and its behaviour would be different depending on the backend that is selected.

Reply:
Yes that would be the goal. If you were referring to the use of the original camel case RunGraphs rather than run_graphs, the naming convention in PyRDF follows PEP8 for now that's why I didn't keep it as the original